File: 250124/물의 상태/state-of-water.c
#include <stdio.h>

int main() {
    int n;
    scanf("%d", &n);
    if(n<0){
        printf("ice");
    }else if(n>=100){
        printf("vapor");
    }else if(0<=n<100){
        printf("water");
    }
    return 0;
}

File: 250625/물의 상태/state-of-water.js
const fs = require("fs");
let input = fs.readFileSync(0).toString();
let n = Number(input);
let temp;
if(n<0){
    temp = 'ice';
}
else if(n>=100){
    temp = 'vapor';
}
else if(0<n<100){
    temp = 'water';
}
console.log(temp);